content/en/docs/6.internals-glossary/1.context.md

```diff
@@ -29,7 +29,7 @@ function (context) {
   } = context
   // Server-side
   if (process.server) {
-    const { req, res, beforeNuxtRender } = context
+    const { req, res, beforeSerialize } = context
```

Comment:
In the examples below, we are referencing to `beforeNuxtRender` maybe keeping it here?